tier4_external_api_msgs/srv/CalibrationDataRequest.srv

```diff
@@ -0,0 +1,5 @@
+bool data_request
```

Comment:
What is this data used for? If it is not needed, remove this line like the [Trigger.srv](https://github.com/ros2/common_interfaces/blob/master/std_srvs/srv/Trigger.srv).